mainapp/templates/nav.html

```diff
@@ -49,6 +49,11 @@
                 <ul class="navbar-nav ml-auto">
                     <!-- right aligned items -->
                     <li class="nav-item active">
+                    {% if request.user.is_authenticated %}
```

Comment:
perhaps it would also be good here to check if the user is a Renter.

Reply:
Great idea! Added.